speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/racescreens/raceselectmenu.cpp

```diff
@@ -1,27 +1,27 @@
 /***************************************************************************
 
-    file        : raceselectmenu.cpp
-    created     : Sat Nov 16 09:36:29 CET 2002
-    copyright   : (C) 2002 by Eric Espie
-    email       : eric.espie@torcs.org   
-    version     : $Id: raceselectmenu.cpp 7011 2020-04-30 15:23:52Z iobyte $                                  
+	file        : raceselectmenu.cpp
+	created     : Sat Nov 16 09:36:29 CET 2002
+	copyright   : (C) 2002 by Eric Espie
+	email       : eric.espie@torcs.org
+	version     : $Id: raceselectmenu.cpp 7011 2020-04-30 15:23:52Z iobyte $
 
  ***************************************************************************/
 
-/***************************************************************************
- *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2 of the License, or     *
- *   (at your option) any later version.                                   *
- *                                                                         *
- ***************************************************************************/
-
-/** @file   
-    		Race selection menu
-    @author	<a href=mailto:eric.espie@torcs.org>Eric Espie</a>
-    @version	$Id: raceselectmenu.cpp 7011 2020-04-30 15:23:52Z iobyte $
-*/
+ /***************************************************************************
+  *                                                                         *
+  *   This program is free software; you can redistribute it and/or modify  *
+  *   it under the terms of the GNU General Public License as published by  *
+  *   the Free Software Foundation; either version 2 of the License, or     *
+  *   (at your option) any later version.                                   *
+  *                                                                         *
+  ***************************************************************************/
+
+  /** @file
+			  Race selection menu
+	  @author	<a href=mailto:eric.espie@torcs.org>Eric Espie</a>
+	  @version	$Id: raceselectmenu.cpp 7011 2020-04-30 15:23:52Z iobyte $
+  */
```

Comment:
Don't run the automatic linter on a speed-dreams file, revert this and then add your changes.